test/analysis/SmartDetectorsAnalysisTests/AlertPresentationTests.cs

```diff
@@ -263,17 +263,19 @@ public PresentationTestAlert(string title = "AlertTitle", ResourceIdentifier res
 
         public class TableData
         {
-            [JsonProperty("prop1")]
-            [TableColumn("First Prop")]
-            public string Prop1 { get; set; }
-
-            [TableColumn("Second Prop")]
+            [TableColumn("Second Prop", Order = 2)]
             public string Prop2 { get; set; }
 
             [UrlFormatter("Link for {NonDisplayProp}")]
             [TableColumn("Uri Prop")]
             public Uri UriProp { get; set; }
```

Comment:
add another prop after Prop1 with no order - make sure it gets placed after UriProp